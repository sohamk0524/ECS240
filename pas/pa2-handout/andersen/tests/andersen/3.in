c Test case with a sequence of address-of and one statement with multiple-dereference on RHS
p 7 6
s 0 2 -1 3
s 0 3 -1 4
s 0 4 -1 5
s 0 5 -1 6
s 0 6 -1 7
s 0 1 4 2
