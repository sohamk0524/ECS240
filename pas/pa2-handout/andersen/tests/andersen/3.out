pt 1 7
pt 2 3
pt 3 4
pt 4 5
pt 5 6
pt 6 7
